src/components/ppeItems/index.js

```diff
@@ -0,0 +1,97 @@
+import React, { useState } from 'react'
+import styled from 'styled-components'
+import { colors } from 'config'
+import { Row, Col } from 'react-bootstrap'
+import CustomModal from './CustomModal'
+
+const PPEItems = (props) => {
+  const { items } = props
+  const [showModal, setShowModal] = useState(0)
```

Comment:
Instead of a number can we use true/false?

Reply:
This is for indexing of each item. so 0 is indicate to no selector